content/brazilian-portuguese/java-basics/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Java: Fundamentos"
+description: "Uma introdução aos fundamentos da linguagem de programação Java"
+date: 2021-10-13T00:00:00Z
+prereq: "Nenhum"
+difficulty: "Iniciante"
+icon: "fab fa-java"
+draft: false
+---
+
+<p style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/udEaocEYlpc" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
+
+Bem vindo ao curso <b>Java: Fundamentos</b>, onde você vai aprender uma linguagem de programação amplamente utilizada e de alto nível, que fornece ao computador instruções para criar programas emocionantes!
+
+Java é uma linguagem de programação de alto nível desenvolvida por James Gosling e Sun Microsystems na Califórnia em 1995. 
+
+<img src="images/logo.png" height="150" alt="Java Logo of steaming coffee cup with Java written below" />
+
+{{% notice info %}}
+Uma linguagem de programação de alto nível é chamada de alto nível porque está a poucos passos do código original executado na Unidade Central de Processamento (CPU) de um computador. O código de alto nível deve ser usado por humanos e é muito mais fácil de entender do que o código de baixo nível. O código de alto nível é posteriormente traduzido para uma linguagem de baixo nível, que só pode ser entendida e executada por uma CPU específica. Para que o código funcione, cada linha de código escrita em uma linguagem de alto nível deve ser traduzida para linguagem de máquina antes de ser implementada pelo computador.
+{{% /notice %}}
+
+Hoje, Java é usado por desenvolvedores em todo o mundo. Especificamente, Java é usado para aplicativos móveis Android, aplicativos de desktop, aplicativos da web, desenvolvimento de jogos 
+
+E, neste curso, abordaremos todos os recursos básicos da linguagem.
+
+![Cartoon, yellow duck named Patrick playing a pink trumpet](https://media.giphy.com/media/l49JKwmJLChtS6d44/giphy.gif) 
```

Comment:
This is also alt text, please translate

i.e translate Cartoon, yellow duck named Patrick playing a pink trumpet